example/glue/ConfigurationDelegate.hpp

```diff
@@ -175,6 +175,15 @@ class MM_ConfigurationDelegate
 	 */
 	MM_GCPolicy getGCPolicy() { return _gcPolicy; }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * Reinitialize the language specific configuration structures.
+	 *
+	 * @return boolean indicating whether configuration reinitialization was successful.
```

Comment:
input parameter description is missing, and there should be no period at the end of `@param/@return` statements.
```suggestion
	 * @return boolean indicating whether configuration reinitialization was successful
```